src/components/FileHistory/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Query } from 'cozy-client'
+
+import HistoryModal from './HistoryModal'
+
+const FileHistory = ({ params: { fileId } }) => {
+  return (
+    <Query query={client => client.get('io.cozy.files', fileId)}>
+      {({ data: file, fetchStatus: fileFetchStatus }) => {
+        return (
+          <Query
```

Comment:
I know we talked about it, but remind me why we can't use `queryConnect` in this case?

Reply:
Why ? Is this because it the querySpec returned by Query.props changes, it is not resubmitted ?